lua/precognition/init.lua

```diff
@@ -80,6 +81,36 @@ local default = {
         PrevParagraph = { text = "{", prio = 8 },
         NextParagraph = { text = "}", prio = 8 },
     },
+    disabled_fts = {
+        "NvimTree",
+        "TelescopePrompt",
+        "aerial",
+        "alpha",
+        "checkhealth",
+        "dapui*",
+        "Diffview*",
+        "Dressing*",
+        "help",
+        "httpResult",
+        "lazy",
+        "lspinfo",
+        "Neogit*",
+        "mason",
+        "neotest%-summary",
+        "minifiles",
+        "neo%-tree*",
+        "netrw",
+        "noice",
+        "notify",
+        "prompt",
+        "qf",
+        "query",
+        "oil",
+        "undotree",
+        "trouble",
+        "Trouble",
+        "fugitive",
```

Comment:
I would rather be more targetted.

When you opened the issue what FT specifically were you having issues with?

Reply:
I had issues with `startify`. Alternatively it could be left as an empty table and add a note on the readme saying that users can add whatever they want?

Reply:
I am happy to implement this for just startify at the moment, and then we can expand as issues come in
From looking at the list, I already use undotree, trouble, fugitive, Lazy, Mason and the existing check catches those

Reply:
Sounds good. I'll update the table and readme accordingly.